package/MDAnalysis/coordinates/DCD.py

```diff
@@ -472,7 +478,7 @@ def _read_next_timestep(self, ts=None):
         """
         if ts is None:
             ts = self.ts
-        ts._frame = self._read_next_frame(ts._x, ts._y, ts._z, ts._unitcell, self.skip)
+        ts._frame = self._read_next_frame(ts._x, ts._y, ts._z, ts._unitcell, 1)
```

Comment:
Yes, document it, add a note to CHANGELOG (even though this seems so obscure that no one _probably_ used it...).